Code/Tools/ProjectManager/Resources/ProjectManager.qss

```diff
@@ -232,6 +232,11 @@ QTabBar::tab:focus {
                             stop: 0 #555555, stop: 1.0 #777777);
 }
 
+#dialogSubTitle {
+    font-size:14px;
```

Comment:
Is this the size of the URL text?  If so, I wonder if it could be a big bigger - maybe 16px?

Reply:
+1 to this, the hyperlink text does look rather small in the screenshot

Reply:
That is not the font-size for the url I believe it is set to 10px right but I'll bump that up